templates/vue/src/components/tyde/activities/CircleOfSupport/CommunityView/AddCommunityForm.vue

```diff
@@ -77,9 +94,16 @@ export default {
       showPicker: false,
       isLoading: false,
       isInputTouched: false,
+      emojiImg: null,
```

Comment:
I believe this is obsolete in which case it should be removed.

Reply:
Removed